### PR TITLE
Make tags property required

### PR DESCRIPTION
**Pull request by thatportugueseguy**
#### This resolves Clubhouse issue: [Make sure that the tags property is always required even though it may be empty](https://app.clubhouse.io/condenastinternational/story/11805/make-sure-that-the-tags-property-is-always-required-even-though-it-may-be-empty)

- [x] Checked that the new code runs properly
- [x] Any schema changes have been added to the change log
- [x] Any schema changes are reflected in the Integration API docs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+
+* Make `tags` property required, even when there are no tags.
```
Comment (srilq) [inline CHANGELOG.md]: Can we word this as a fix so people don't think this is an intentional schema change. (e.g. ``Fix `tags` being optional.``)

File: models/article.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ module.exports = articleBaseModel.keys({
     sections: joi.array().items(categoryModel).length(1).required()
   }).required(),
   // tags
-  tags: joi.array().items(joi.string()).optional(),
+  tags: joi.array().items(joi.string()).required(),
```
Comment (srilq) [inline models/article.js]: LGTM, assuming this will allow an empty array.

File: CHANGELOG.md (hunk 1 above)
Comment (thatportugueseguy) [reply CHANGELOG.md]: 👍

File: models/article.js (hunk 2 above)
Comment (thatportugueseguy) [reply models/article.js]: It does, i tested.

**Pull request merged**